locale/en/LC_MESSAGES/django.po

```diff
@@ -24,13 +24,13 @@ msgstr ""
 #: forms.py:49
 #, python-format
 msgid ""
-"The \"%s\" are not a Speakerfight user. For now, we just allow already "
-"joined users."
+"The \"%s\" is not a Speakerfight user. For now, we only allow already "
```

Comment:
skip the "**The**" before the actual name

also can skip the redundant "**already**" from the sentence.